### DISCUSSION TITLE
is a pageloader that uses setTimeout() penalizing the score?

### DISCUSSION THREAD

**Discussion by koraysels** [GoogleChrome/lighthouse#13577] [Q&A]
I have a website that has a page loader animation for 2.4 seconds.. 
is that time responsible for the insanely bad performance score ?

![image](https://user-images.githubusercontent.com/2418853/149829317-075123b0-0840-4583-9417-c4f431ba92a1.png)

**Comment by huseyintasdemir1** [comment]
Hi Koray,
Can you share the your site address so we can try to help?

You can also ask these forums for other Google webmaster topics.
English :  https://support.google.com/webmasters/community?hl=en
Turkish :  https://support.google.com/webmasters/community?hl=tr

**Comment by koraysels** [comment]
Sure! Please don't ddos ;) 

[https://ellenverbiest.be](https://ellenverbiest.be)

**Comment by huseyintasdemir1** [comment]
Do not worry :)
There are many errors.
Especially "Total Blocking Time" first check this one.
https://web.dev/lighthouse-total-blocking-time/

**Comment by koraysels** [comment]
I don't understand what you mean...

**Comment by adamraine** [comment]
It's not single handedly bringing down your performance score, but it does appear to be a factor. Lighthouse actually considers the page to be loaded before any of the images show up, so it's only considering the animation on the title element.

The animation on your title element is affecting the LCP. The LCP event is at the end of the animation that changes the size of the underline. If I manually remove that animation, the LCP event moves to the end of the fade-in animation. This is probably WAI, but is related to https://github.com/GoogleChrome/lighthouse/issues/10869.

The high TBT, TTI, and CLS seem unrelated to the animation but have a [high impact on the performance score](https://googlechrome.github.io/lighthouse/scorecalc/#FCP=2020&SI=4612&LCP=15657&TTI=10348&TBT=1689&CLS=0.57&FMP=2021&device=mobile&version=8.5.0).
